handler/handler.go

```diff
@@ -623,39 +651,69 @@ func score(category yahtzee.Category, dices []int) (int, error) {
 	s := 0
 	switch category {
 	case yahtzee.Ones:
+		dicesConsidered := 0
 		for _, d := range dices {
+			if dicesConsidered == 5 {
+				break
+			}
 			if d == 1 {
 				s++
+				dicesConsidered++
```

Comment:
instead of this "dicesConsidered" you could use the previous way of counting dices with the right face then set `s = max(s, 5 * 1)`